test/tests/extraneous_prints_tests.py

```diff
@@ -22,7 +22,7 @@ def test_for_extra_prints(self):
             "grep",
             "-R",
             "-I",
-            '[^f]print('
```

Comment:
Look closer at this test. There's a way to note when pprint and print statements are acceptable via comments.